src/bar/bar.f90

```diff
@@ -1,6 +1,9 @@
-Subroutine bar(a, b)
+module bar_module
+  contains
+  Subroutine bar(a, b)
+    !DIR$ ATTRIBUTES DEFAULT, DLLEXPORT, DECORATE, ALIAS: 'bar' :: bar
```

Comment:
I've been out of the Fortran game for a while. What is this directive?

Reply:
This is an Intel compiler directive, with the specific attributes needed to create a Windows DLL.  I wanted to demonstrate the ability to create a DLL with this example.  I realize this set of directives is a compiler-specific thing, but the syntax should be recognized as a comment by other non-Intel compilers.  I'm open to adding examples from other compilers if folks can provide those.